src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -193,17 +193,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 				// Convert bytestream into BufferedImage object
 				BufferedImage image = ImageIO.read(fileStream);
 
-				// Create image attachment object from BufferedImage object
-				ImageAttachment imageAttachment = new ImageAttachment(UUID.randomUUID(), image);
-
-				// Set the content of the message to an img tag that calls the ImageServlet
-				String src = "/image/" + imageAttachment.getId().toString();
-				String content = "<img src=" + src + " width=500>";
-				Message message = new Message(UUID.randomUUID(), conversation.getId(),
-						user.getId(), content, Instant.now());
-
-				messageStore.addMessage(message);
-				imageStore.addImage(imageAttachment);
+				// Check that the file was actually an image
```

Comment:
I would update either this comment or the one above to mention that ImageIO.read() will return null if the file cannot be processed as an image.